RELEASE.md

```diff
@@ -0,0 +1,84 @@
+# Release Instructions.
+
+> Note: Versions should be semantic versioning maj.min.patch style, except for the default
+> HEAD-SNAPSHOT which is the version used on the `main` branch
+
+Preconditions:
+
+  1. You must have a username on oss.sonatype.org.
+    - If you need one, sign up at issues.sonatype.org.
+  2. Your sonatype identity must have permissions to publish to org.squareup.
+    - Fill out a [ticket] and tag in a square opensource contributor who already has access.
+  3. You must have a PGP private key (typically using GnuPG) whose key you have uploaded
+     to the main keyservers.
+     - Follow the instructions here: https://central.sonatype.org/pages/working-with-pgp-signatures.html
+     - Pay especial attention to https://central.sonatype.org/pages/working-with-pgp-signatures.html#distributing-your-public-key
+     - use `gpg --list-keys` to see the keys available.
+  4. Obviously have gpg, bazelisk, java, and other preconditions of building installed.
+
+Steps:
+  1. Prepare the repo
+      1. `git clone` repository or update with `git fetch`
+      2. `git checkout origin/main`
+      3. `git switch -c release-<version>` # create a release branch
+
+  2. Prepare the release artifacts (and test the system)
+      1. Edit the `versions.bzl` file updating the LIBRARY_VERSION var to the release version.
+      2. `git commit -m "Prepare to release version <version>" versions.bzl`
+      3. `git tag v<version>` # version such as v0.0.6. Include the v. 
+      4. `bazel build //... && bazel test //...`
+
+  3. Deploy the artifacts
+      1. ```
+          tools/deploy.kts \
+             --key <yourgpgkey> \
+             --username <yourusername> \
+             --password <yourpassword> \
+         ```
+          - username/password are for oss.sonatype.org
+      2. Enter your gnupg passphrase for the given PGP key (if you set a passphrase for it).
+      3. Wait for the deployment to push all the files to sonatype. 
+      4. Log in to `oss.sonatype.org`
+      5. Search for "staging repositories"
+      6. Find a "comsquareup-<somenumber>" repo.
```

Comment:
comsquareup is right right?

Reply:
it is right. The repo names are temporary, and it's just a mnemonic in their UI, so it's just the first bits of the groupID squashed together.

Reply:
But... that needs backticks, not quotes, so `<somenumber>` isn't treated as html.